decimal64.go

```diff
@@ -389,8 +382,8 @@ func (d Decimal64) Signbit() bool {
 
 // Class returns a string of the 'type' that the decimal is.
 func (d Decimal64) Class() string {
-	dp := d.getParts()
-
+	dp := decParts{}
+	dp.unpack(d)
```

Comment:
This syntax is used because it's quicker than returning a struct or using even using an inline function